tests/schemas/api_v3/block.schema.js

```diff
@@ -14,17 +14,22 @@
  *
  */
 import Joi from 'joi';
+import regex from './regex';
+
+const generator = {
+	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
```

Comment:
Is it possible to resolve the generator `publicKey` and `name` to extend this response?

Reply:
Yes, it is already added. Just need to add those in test schemas.